### PR TITLE
LOC CHECKIN | dotnet/cli release/2.2.2xx | 20190215

### PR DESCRIPTION
**Pull request by v-zbsail**
- If this PR should not run tests please add text "skip[REMOVE_THIS]ci[REMOVE_THIS]please" (remove the marked text, no quotes).
- Please add description for changes you are making.
- If there is an issue related to this PR, please add the reference.

### CODE REVIEW

File: src/dotnet/CommandLine/xlf/LocalizableStrings.es.xlf [hunk 1]
```diff
@@ -79,7 +79,7 @@
       </trans-unit>
       <trans-unit id="UsageCommandArgs">
         <source> [args]</source>
-        <target state="translated"> [args]</target>
+        <target state="translated">[args]</target>
```
Comment (nguerrera) [inline src/dotnet/CommandLine/xlf/LocalizableStrings.es.xlf]: @v-zbsail Leading space seems to have been stripped from translation here.

File: src/dotnet/ToolManifest/xlf/LocalizableStrings.cs.xlf [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="cs" original="../LocalizableStrings.resx">
+    <body>
+      <trans-unit id="CannotFindPackageIdInManifest">
```
Comment (nguerrera) [inline src/dotnet/ToolManifest/xlf/LocalizableStrings.cs.xlf]: This file looks like it came from a branch ahead of release/2.2.2xx!

File: src/Microsoft.DotNet.Cli.Utils/xlf/LocalizableStrings.zh-Hans.xlf [hunk 3]
```diff
@@ -141,7 +141,7 @@
       </trans-unit>
       <trans-unit id="UnableToGenerateDepsJson">
         <source>Unable to generate deps.json, it may have been already generated.  You can specify the "-d" option before the tool name for diagnostic output (for example, "dotnet -d &lt;toolname&gt;": {0}</source>
-        <target state="translated">无法生成 deps.json，它可能已经生成。可在诊断输出的工具名称之前指定 "-d" 选项（例如，"dotnet -d &lt;toolname&gt;"）: {0}</target>
+        <target state="translated">无法生成 deps. json，它可能已生成。可在诊断输出的工具名称前面指定 "-d" 选项(例如，"dotnet -d &lt;toolname&gt;": {0}</target>
```
Comment (nguerrera) [inline src/Microsoft.DotNet.Cli.Utils/xlf/LocalizableStrings.zh-Hans.xlf]: whitespace after deps.json is suspect

File: src/dotnet/CommandLine/xlf/LocalizableStrings.es.xlf (hunk 1 above)
Comment (nguerrera) [reply src/dotnet/CommandLine/xlf/LocalizableStrings.es.xlf]: This happened a bunch.

**Comment by nguerrera** [comment]
@dotnet-bot Test Linux arm64 Debug Build
@dotnet-bot Test fedora.27 x64 Debug Build

**Comment by nguerrera** [comment]
resolver tests have hard-coded english expectations. I will fix this.

**Comment by nguerrera** [comment]
@dotnet-bot Test Linux x64 Release Build

(Same sporadic nuget issue again)

**Comment by tmeschter** [comment]
@livarcocc @nguerrera What's the next step in getting this merged in?

**Comment by nguerrera** [comment]
@tmeschter I pushed a commit to manually put leading spaces back. This is a loc tooling bug tracked internally by https://ceapex.visualstudio.com/CEINTL/_workitems/edit/67751

@livarcocc Are we ok to merge in this branch when green?

**Comment by livarcocc** [comment]
@nguerrera yes.

**Comment by livarcocc** [comment]
merge it ASAP though.

**Comment by nguerrera** [comment]
Found another issue. There was a string moved after the deadline. Recycling the translation manually.

Also need to backport 1 string to 2.1.6xx. Separate PR for that coming separately.

**Comment by tmeschter** [comment]
@nguerrera I really need to make a GitHub App that blocks changes to .xlf files after the cut-off.

**Comment by livarcocc** [comment]
@dotnet-bot Test this please

**Comment by nguerrera** [comment]
@dotnet-bot Test this please

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT arm Debug Build
@dotnet-bot Test Windows_NT x64 Release Build
@dotnet-bot Test Windows_NT x86 Debug Build
@dotnet-bot Test Windows_NT_ES x64 Debug Build
@dotnet-bot Test Windows_NT_NoSuffix x64 Release Build

**Pull request merged**